### PR TITLE
Remove some obsolete jcl preprocessor configurations

### PR DESCRIPTION
**Pull request by pshipton**
Removed
  SIDECAR19-SE-B165
  SIDECAR19-SE-B169
  SIDECAR19-SE-B174
  SIDECAR19-SE-B175

Depends on ibmruntimes/openj9-openjdk-jdk9#127

Signed-off-by: Peter Shipton <Peter_Shipton@ca.ibm.com>

### CODE REVIEW

**Comment by pshipton** [comment]
jenkins test sanity plinux jdk9 depends ibmruntimes/openj9-openjdk-jdk9#127

**Comment by pshipton** [comment]
The PR testing had passed, but I'll run it again since I rebased.

jenkins test sanity plinux jdk9 depends ibmruntimes/openj9-openjdk-jdk9#127

File: jcl/src/java.base/share/classes/java/lang/invoke/VarHandleGuards.java [hunk 1]
```diff
@@ -1,7 +1,7 @@
-/*[INCLUDE-IF Sidecar19-SE-B174]*/
+/*[INCLUDE-IF Sidecar19-SE]*/
```
Comment (DanHeidinga) [inline jcl/src/java.base/share/classes/java/lang/invoke/VarHandleGuards.java]: Shouldn't this be `Sidecar19-SE-OpenJ9`?  Here and below

Comment (pshipton) [reply jcl/src/java.base/share/classes/java/lang/invoke/VarHandleGuards.java]: Technically yes, but using Sidecar19-SE doesn't break anything. I expect Sidecar19-SE-OpenJ9 to be replaced with Sidecar19-SE once SIDECAR19-SE-B148 is deleted, so using Sidecar19-SE now saves some changes later.

**Pull request merged**